cpg_workflows/stages/seqr_loader_long_read/long_read_snps_indels_annotation.py

```diff
@@ -46,9 +45,28 @@
 }""",
 )
 
+LRS_IDS_QUERY = gql(
+    """
+    query MyQuery($dataset: String!) {
+    project(name: $dataset) {
+      sequencingGroups(technology: {eq: "long-read"}, type: {eq: "genome"}) {
+          id
+          sample {
+            externalId
+            meta
+            participant {
+              externalId
+            }
+          }
+        }
+      }
+    }
+    """,
+)
+
 
-@cache
-def query_for_snps_indels_vcfs(dataset_name: str) -> dict[str, str]:
+@lru_cache
```

Comment:
bit of a nit, but can you revert this to `from functools import cache` (or lru_cache). the naked `cache` is just a lru_cache with no specified max size